### PR TITLE
Local Image upload and ui change to recipe and home page

### PR DESCRIPTION
**Pull request by Tpanyasr**

### CODE REVIEW

File: frontend/src/components/Form.js [hunk 1]
```diff
@@ -1,55 +1,85 @@
...
     steps: [],
-    user: ""
+    user: "",
   });
 
+  useEffect(() => {
+    cloudinaryRef.current = window.cloudinary;
+    widgetRef.current = cloudinaryRef.current.createUploadWidget(
+      {
+        cloudName: "dype2qwpo",
```
Comment (crowdedpoem) [inline frontend/src/components/Form.js:33]: Should this be an environment variable .env

File: package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "prettier": "2.8.0"
```
Comment (crowdedpoem) [inline package.json:3]: Why is prettier outside of the frontend and backend folders?

**Review by crowdedpoem — COMMENTED** [review]
just a couple questions, but it looks good!

**Review by tylerboo — APPROVED** [review]
Nice!

File: package.json (hunk 2 above)
Comment (Tpanyasr) [reply package.json:3]: Good question it's outside so that it can be run on both folders for style formatting

**Review by crowdedpoem — APPROVED** [review]

File: frontend/src/components/Form.js (hunk 1 above)
Comment (Tpanyasr) [reply frontend/src/components/Form.js:33]: I'll try to add this change

**Pull request merged**